sdks/builds/android.mk

```diff
@@ -2,7 +2,10 @@
 ANDROID_URI?=https://dl.google.com/android/repository/
 ANT_URI?=https://archive.apache.org/dist/ant/binaries/
 
+ANDROID_CMAKE_VERSION?="3.6.4111459"
+ANDROID_SDK_PREFIX?=$(ANDROID_TOOLCHAIN_DIR)/sdk
```

Comment:
Is anything going to override this? Remember that XA will not be able to override it whenever it's going to use the Mono Archive.

Reply:
The version may change if the Android SDK updates it. It's just a convenient location for the variable part of the module name.